src/aof.c

```diff
@@ -1126,14 +1126,16 @@ void flushAppendOnlyFile(int force) {
         usleep(server.aof_flush_sleep);
     }
 
-    latencyStartMonitor(latency);
+    durationStartMonitor(latency);
     nwritten = aofWrite(server.aof_fd,server.aof_buf,sdslen(server.aof_buf));
-    latencyEndMonitor(latency);
+    durationEndMonitor(latency);
+    durationAddSample("aof", latency, 0);
     /* We want to capture different events for delayed writes:
      * when the delay happens with a pending fsync, or with a saving child
      * active, and when the above two conditions are missing.
      * We also use an additional event name to save all samples which is
      * useful for graphing / monitoring purposes. */
+    latency /= 1000;
```

Comment:
Forget to remove?

Reply:
Above this line, `durationStartMonitor` assigned its value in microseconds, but below this line, `latencyAddSampleIfNeeded` takes value in milliseconds.